packages/documentation/src/stories/components/elements/paragraph/paragraph.stories.ts

```diff
@@ -0,0 +1,34 @@
+import type { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { MetaExtended } from '@root/types';
+
+const meta: MetaExtended = {
+  id: '7ecd87f1-de96-4e39-a057-ba1798eb69593',
+  title: 'Elements/Paragraph',
```

Comment:
Not sure we want to document elements, let's discuss this during a dev roundtable.

Reply:
I just opened a ticket for that.

Reply:
@bashirkarimi Could you update the PR against this resolution? https://github.com/swisspost/design-system/issues/3584#issuecomment-2376751001